src/APIs/Billing.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+namespace habil\ResellerClub\APIs;
+
+use habil\ResellerClub\Helper;
+use SimpleXMLElement;
+
+/**
+ * Class Billing
+ *
+ * @package habil\ResellerClub\APIs
+ */
+class Billing
+{
+    use Helper;
+
+    /**
+     * @var string
+     */
+    protected $api = 'billing';
+
+    /**
+     * Gets the transaction details for the specified Customer Id.
+     *
+     * @param $customerId
+     *
+     * @return mixed|SimpleXMLElement
+     */
+    public function customerTransactions($customerId)
+    {
+        return $this->get('customer-transactions', ['customer-id' => $customerId, 'transaction-ids' => 99210620]);
```

Comment:
I think you left some hardcoded values here: 
```'transaction-ids' => 99210620```

Reply:
Always check twice :)